### ISSUE TITLE
Accessibility finding: Missing focus management on profile pages

### ISSUE THREAD

**Issue by emisparkles** [department-of-veterans-affairs/va.gov-team#133950]
### Checklist item

WEB-243	Tab order follows a logical sequence

### Severity level

3, Medium. Should be fixed in 1-3 sprints post-launch.

### Page(s)/url(s)

Profile > Direct deposit information
https://staging.va.gov/profile/direct-deposit

Profile > Email and text notifications > Update email/mobile phone subtask flow
https://staging.va.gov/profile/edit?fieldName=email&returnPath=%2Fprofile%2Fnotifications
https://staging.va.gov/profile/edit?fieldName=mobilePhone&returnPath=%2Fprofile%2Fnotifications

### Details

When clicking the Edit button on the direct deposit form, or when updating contact information in the notifications settings subtask flow, focus is lost. The page heading is announced, but focus is set to `body.merger`.

### Reproduction steps

1. With a screen reader active, log into staging.va.gov
2. Open the browser console in devtools
3. Navigate to profile/direct-deposit
4. Click Edit to open the form
5. Screen reader will announce the page heading
6. Type `document.activeElement` into the browser console
7. Observe output is `body.merger`
8. Navigate to profile/notifications
9. Click "Update your email address" or "Update your mobile phone number"
10. Repeat steps 5-7

### Proposed solution or next steps

Programmatically set focus to the first form element in the direct deposits page by adding focus management to [AccountInfoView.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/7a3622cdee9b2c86a33a38ef42732cc52cde92eb/src/applications/personalization/profile/components/direct-deposit/AccountInfoView.jsx#L4) and [AccountUpdateView.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/7a3622cdee9b2c86a33a38ef42732cc52cde92eb/src/applications/personalization/profile/components/direct-deposit/AccountUpdateView.jsx).
Programmatically set focus to the page heading for the update contact info subflow from the notifications page by updating [Edit.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/7a3622cdee9b2c86a33a38ef42732cc52cde92eb/src/applications/personalization/profile/components/edit/Edit.jsx#L152-L167).
